### PR TITLE
Add NoteScreenPlugin to Fudy. Still need to refactor

### PR DESCRIPTION
**Pull request by DongTrieu1998**
Add NoteScreenPlugin into project. Still need to refactor

### CODE REVIEW

**Comment by DongTrieu1998** [comment]
Hi @thanhdat369 , I add a QML plugin into project. It still has something need to be improved but could you review it? Thank you.

File: fudy/HomeScreen.qml [hunk 1]
```diff
@@ -50,7 +50,7 @@ Item {
 				onClicked: {
 					switch(model.name)
 					{
-					case "NOTE" : internal.openApplication("qrc:/NoteScreen.qml"); break
+					case "NOTE" : internal.openApplication("D:/TEST/fudy_project/fudy/NoteScreen/my/plugin/example/NoteScreen.qml"); break
```
Comment (ghost) [inline fudy/HomeScreen.qml:53]: It's risk, please create ticket to refactor it

File: fudy/NoteScreen/my/plugin/example/qmldir [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# Plugin name.
+# Make sure that is the same name as your library target in
+# CMakeLists.txt
+plugin NoteScreen
+
+# Resourse files
+# 'qmldir' syntax for resources is:
+# <ResourceIdentifier> <InitialVersion> <File>
+# With 'qrc:' you can tell Qt to look for files in memory and it's
```
Comment (ghost) [inline fudy/NoteScreen/my/plugin/example/qmldir:17]: Remove all comment

File: fudy/main.cpp [hunk 3]
```diff
@@ -2,6 +2,7 @@
 #include <QQmlApplicationEngine>
 #include "StylingComponents/myColor.h"
 #include "StylingComponents/myFont.h"
+#include <QDir>
```
Comment (ghost) [inline fudy/main.cpp:5]: Is it neccessary?

**Review by ghost — APPROVED** [review]

**Pull request merged**